src/overtone/sc/sclang.clj

```diff
@@ -245,7 +246,7 @@
 
 (defn- check-proc!
   [proc args]
-  (loop [counter 15]
+  (loop [counter 100]
```

Comment:
If the code has some syntax error, it will take too long for the REPL to return. Wdyt of making it configurable by a dynamic or atom or some other form of configuration?